src/BinSkim.Sdk/BinaryAnalyzerContext.cs

```diff
@@ -79,6 +79,8 @@ public CompilerDataLogger CompilerDataLogger
 
         public bool IgnorePdbLoadError { get; set; }
 
+        public bool IncludeWixBinaries { get; set; }
```

Comment:
>IncludeWixBinaries [](http://example.com/codeflow?start=20&length=18)

You didn't add a PerLanguageOption for this, as a result no one can configure this property via XML configuration.

Reply:
fixed